### PR TITLE
UXarray Themed Colormaps

### LINKED ISSUE #786: UXarray Themed Colormap
It would be nice to have a colormap based off the color's from the UXarray logo to use for examples.

[image](file:///private/tmp/mydocs/_static/uxarray_temp_logo.png)

Both Sequential and Diverging versions would be great to have. 

https://matplotlib.org/stable/users/explain/colors/colormaps.html

### PR DESCRIPTION
**Pull request by aaronzedwick**
Closes #786 

## Overview

Adds two UXarray-styled colormaps: diverging and sequential.
Diverging:
image
Sequential:
image

## Expected Usage

```Python
import uxarray as ux

uxds = ux.open_dataset(grid_path, data_path)

uxds[v1].plot(cmap=ux.cmaps.sequential, backend='matplotlib')

uxds[v1].plot(cmap=ux.cmaps.diverging, backend='matplotlib')

```

## PR Checklist

**General**
- [x] An issue is linked created and linked
- [x] Add appropriate labels
- [x] Filled out Overview and Expected Usage (if applicable) sections

### CODE REVIEW

File: uxarray/cmaps.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        (
+            (0.000, (0.698, 0.463, 0.231)),
+            (0.400, (0.867, 0.678, 0.498)),
+            (0.500, (1.000, 1.000, 1.000)),
+            (0.600, (0.757, 0.859, 0.541)),
+            (1.000, (0.200, 0.235, 0.125)),
+        ),
+    )
+
+    return diverging_map
```
Comment (philipc2) [inline uxarray/cmaps.py:26]: I believe these should be stored as variables so they can be accessed as `uxarray.cmaps.sequential`

**Review by philipc2 — CHANGES_REQUESTED** [review]

**Comment by philipc2** [comment]
Also from the usage example in the overview, it should be something like the following:

```Python
import uxarray as ux

uxds[v1].plot(cmap=ux.cmaps.sequential, backend='matplotlib')

uxds[v1].plot(cmap=ux.cmaps.diverging, backend='matplotlib')

```

**Comment by philipc2** [comment]
The addition to the user guide section also appears to be failing.

image

File: uxarray/cmaps.py (hunk 1 above)
Comment (aaronzedwick) [reply uxarray/cmaps.py:26]: Yeah, you're right! That is why it's failing in the docs.

**Comment by aaronzedwick** [comment]
How do you guys think the colors look?

**Review by rajeeja — APPROVED** [review]

**Comment by rajeeja** [comment]
> How do you think the colors look? @rajeeja @philipc2 

👍

**Comment by philipc2** [comment]
@aaronzedwick 

The colormaps look great, but I'm wondering if maybe we would like to consider different colors all together for the style of Uxarray?

We still don't have a true package logo. Would you like to sit down together (including @rajeeja too) to brainstorm some ideas for the theme of Uxarray?

For the text of the logo, I was considering something like the following. 

image

For reference, xarray:

image

It would be nice to put together a style guide for our package, logos, and design. Something like: 

image

**Comment by philipc2** [comment]
I've always liked the idea of this style of logo, which emphasizes the spherical unstructured grid support we provide. 

image

**Comment by philipc2** [comment]
We could also adapt the Project Raijin logo for UXarray, which I think looks quite nice. We could modify the colors too. 

image

**Comment by aaronzedwick** [comment]
> We could also adapt the Project Raijin logo for UXarray, which I think looks quite nice. We could modify the colors too.
> 
> image

I like these colors a lot more than the other ones.

**Comment by philipc2** [comment]
Here are the colors from the new logo if you'd like to update these colormaps:

Blue:
#016691

Green:
#049390

Gray: 
#343741

**Comment by rajeeja** [comment]
> @aaronzedwick
> 
> The colormaps look great, but I'm wondering if maybe we would like to consider different colors all together for the style of Uxarray?
> 
> We still don't have a true package logo. Would you like to sit down together (including @rajeeja too) to brainstorm some ideas for the theme of Uxarray?
> 
> For the text of the logo, I was considering something like the following.
> 
> image
> For reference, xarray:
> 
> image
> It would be nice to put together a style guide for our package, logos, and design. Something like:
> 
> image

+1 let's get this done Friday at UChicago.

**Comment by aaronzedwick** [comment]
@philipc2 I really like how these came out.

image
image
image

**Comment by philipc2** [comment]
> @philipc2 I really like how these came out.
> 
> image image image

These are very pretty! 

One small nitpick: can you make the blue & green ones start at zero (white)?

**Comment by philipc2** [comment]
Can you see how a colormap would look if it's a White -> Green -> Blue sequence? 

A lot of the sequential ones on mpl appear to start with white.

image

**Comment by aaronzedwick** [comment]
> > @philipc2 I really like how these came out.
> > image image image
> 
> These are very pretty!
> 
> One small nitpick: can you make the blue & green ones start at zero (white)?

Oh yeah that would be important!

**Review by philipc2 — CHANGES_REQUESTED** [review]
Everything looks great here! Once we have the following color maps, it'll be ready! 
- Sequential: White, Blue, Green
- Sequential: White, Blue
- Sequential: White, Green
- Diverging: Blue, White, Green

**Comment by aaronzedwick** [comment]
image
What do you wanna call this one then?

**Comment by aaronzedwick** [comment]
> Everything looks great here! Once we have the following color maps, it'll be ready!
> 
> * Sequential: White, Blue, Green
> * Sequential: White, Blue
> * Sequential: White, Green
> * Diverging: Blue, White, Green

The blue looks darker to me so I think it makes more sense to go green-blue with the colors.

**Comment by philipc2** [comment]
image

Can we also add edges to this plot? Maybe a title would be good too for each.

**Review by philipc2 — APPROVED** [review]
These colormaps look great! Thanks for putting them together.

**Pull request merged**